repo/packages/K/kafka/900/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "upgradesFrom": [
+    "2.4.0-1.1.1",
+    "2.5.0-2.1.0",
+    "2.6.0-2.2.1",
+    "2.7.0-2.3.0"
```

Comment:
This is really nice - users can potentially skip 3 versions if they want to get to latest. 👍